Audio-Transcription/workers/WavWorker.js

```diff
@@ -35,7 +35,7 @@ function start(bufferSize) {
     socket.send("handshake");
   };
   socket.onmessage = (event) => {
-    console.log(event.data);
+    self.postMessage({ command: "transcription", text: event.data});
```

Comment:
Could you add this listener to ```Mp3Worker.js``` as well?

Reply:
Nice, catch, done.